Classes/common/CDTReplicator/CDTAbstractReplication.h

```diff
@@ -127,6 +128,15 @@ typedef NS_ENUM(NSInteger, CDTReplicationErrors) {
 */
 @property (nonatomic, copy) NSDictionary* optionalHeaders;
 
+NS_ASSUME_NONNULL_BEGIN
+
+@property (nonnull, nonatomic, readonly, strong) NSArray* httpInterceptors;
```

Comment:
We can remove it, however I think its beneficial to have it spelt out until nullability annotations is norm in Objective-c code.